### ISSUE TITLE
Requests pagination error

### ISSUE THREAD

**Issue by dyanakiev** [tobiasdierich/gauge#4]
Requests pagination error

```
ErrorException
Trying to get property 'content' of non-object (View: /...../vendor/tobiasdierich/gauge/resources/views/components/method-badge.blade.php)
https://....../requests/e0e39d220ff38421b6dd61a998975b28?page=2
```
![image](https://user-images.githubusercontent.com/11967079/102483981-43065200-406e-11eb-82cb-c9bfd44edae9.png)

**Comment by tobiasdierich** [comment]
Thanks for reporting the issue. I'll have a look at it over the course of the weekend.

**Comment by tobiasdierich** [comment]
Fixed in `v0.0.5`
